worlds/witness/options.py

```diff
@@ -142,6 +143,33 @@ class ShuffleEnvironmentalPuzzles(Choice):
     option_obelisk_sides = 2
 
 
+class EasterEggHunt(Choice):
+    """
+    Adds up to 100 Easter Eggs to the game, placed by NewSoupVi, Exempt-Medic, hatkirby, Scipio, and Rever.
+    These can simply be collected by clicking on them.
+
+    Each consecutive check will logically expect 5 Easter Eggs to be collected.
+    However, the amount that you need to collect to actually trigger the next location check differs by option choice:
+    - Easy: For every 2 Easter Eggs collected, an item is sent
+    - Normal: For every 3 Easter Eggs collected, an item is sent
+    - Hard: For every 4 Easter Eggs collected, an item is sent
+    - Very Hard: For every 5 Easter Eggs collected, an item is sent (You are expected to find every Easter Egg)
+
+    On Easy, there will still be a location check for "100 Eggs Collected" (provided there aren't any unreachable eggs due to options like shuffle_postgame: False), but all checks after 40 Eggs will be excluded.

```

Comment:
```suggestion
    On Easy, there will still be a location check for "100 Easter Eggs Collected" (provided there aren't any unreachable eggs due to options like shuffle_postgame: false), but all checks after 40 Eggs will be excluded.
```
The second one is because in a yaml it's usually `false` rather than `False`